exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -158,7 +161,8 @@ func (prwe *PRWExporter) PushMetrics(ctx context.Context, md pdata.Metrics) erro
 			errs = append(errs, exportErrors...)
 		}
 
-		if dropped != 0 {
+		if dropped > 0 {
+			prwe.logger.Info("Prometheus remote write dropped metrics", zap.Int("dropped_count", dropped))
```

Comment:
This can potentially flood the logs. We shouldn't be doing this. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#logging

I believe all exporters record dropped metrics, which presumably is sufficient for observing the Collector. If we need a different way observe dropped data we need to explain why and we would prefer it to be uniform for all exporters, not special just for Prometheus.

Reply:
The metrics would work in this case. I opened the PR due to my lack of awareness of the builtin metrics. I'll close this.